.github/ISSUE_TEMPLATE/2_bug_report_linux.yml

```
@@ -0,0 +1,108 @@
+- type: textarea
+  attributes:
+      label: "Important Information"
+      description: >
+          - mpv version
+           - Linux Distribution and Version
+           - Source of the mpv binary
+           - If known which version of mpv introduced the problem
+           - Window Manager and version
+           - GPU model, driver and version
+           - Possible screenshot or video of visual glitches
+
+          If you're not using git master or the latest release, update.
+          Releases are listed [here](https://github.com/mpv-player/mpv/releases).
+      value: |-
+          - mpv version:
+          - Linux version:
+          - Source:
+          - Introduced in version:
+          - Window Manager and version:
+          - GPU model, driver and version:
+  validations:
+      required: true
+- type: textarea
+  attributes:
+      label: "Reproduction Steps"
+      description: >
+          Try to reproduce your issue with `--no-config` first. If it isn't reproducible
+          with `--no-config` try to find out which option or script causes your issue.
+
+
+          If the issue is render performance-related, try to reproduce it with `--no-config
+          --profile=fast`, which is designed to work on lower-end mobile devices.
+
+          For decoding performance-related issues, additionally try to enable hardware
+          decoding with `--hwdec=yes` or the `Ctrl+h` shortcut.
+
+
+          Describe the reproduction steps as precise as possible. It's very likely that
+          the bug you experience wasn't reproduced by the developer because the workflow
+          differs from your own.
+  validations:
+      required: true
+- type: textarea
+  attributes:
+      label: "Expected Behavior"
+  validations:
+      required: true
+- type: textarea
+  attributes:
+      label: "Actual Behavior"
+  validations:
+      required: true
+- type: textarea
+  attributes:
+      label: "Log File"
+      description: >
+          Make a log file with `--log-file=output.txt`, which is our preferred way, and attach it to
+          the issue.
+
+
+          If you have trouble producing a log file, you can alternatively use `-v -v` and save the
+          terminal output to a file.
+
+
+          In the case of a crash, please provide a backtrace.
+
+
+          Without the log file, this issue will be closed for ignoring the issue template.
+  validations:
+      required: true
+- type: textarea
+  attributes:
+      label: "Sample Files"
+      description: >
+          Please provide sample files for a minimal reproduction case, if the issue can't be
+          reproduced without it or it being helpful to demonstrate the issue.
+
+
+          Files can be attached directly within the text field below. If github's upload restriction
+          (file type, size, etc) don't allow the direct upload, use a solutions that is easy to
+          access with possible long-term retention. Possible service can be but are not limited to:
```

```suggestion
          Preferably files should be attached directly within the text field below. Due to github's upload restriction, you may need to archive the sample file first and upload `.zip` file. If sample file is too big to attach to the issue directly, use a solutions that is easy to access with possible long-term retention.
          Possible service can be but are not limited to:
```
Not formatted, but main point is to mention `.zip` upload. Direct upload to issue is always better, than external hosting.